### PR TITLE
relax action* dependencies

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
At least according to the tests (ran locally) this is still compatible with 6.0.0beta1, so relax the dependencies to make it possible for Rails 6.x to still use this gem.

**Comment by nickcoyne** [comment]
Is anyone monitoring this repo? Could we get this merged?

File: record_tag_helper.gemspec [hunk 1]
```diff
@@ -15,10 +15,10 @@ Gem::Specification.new do |spec|
 
   spec.files = Dir['CHANGELOG.md', 'LICENSE', 'README.md', 'lib/**/*']
 
-  spec.add_dependency 'actionview', '~> 5.x'
+  spec.add_dependency 'actionview', '>= 5', '< 7'
```
Comment (rafaelfranca) [inline record_tag_helper.gemspec]: Can you remove the `< 7`?

File: .travis.yml [hunk 2]
```diff
@@ -3,7 +3,9 @@ language: ruby
 sudo: false
 
 rvm:
-  - 2.2.2
```
Comment (rafaelfranca) [inline .travis.yml:6]: Can we keep support to 2.2?

**Review by rafaelfranca — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (mmoll) [reply .travis.yml:6]: Sure, I added the needed pinnings to the Gemfile.

File: record_tag_helper.gemspec (hunk 1 above)
Comment (mmoll) [reply record_tag_helper.gemspec]: Done.

**Comment by mmoll** [comment]
How should `Gemfile.lock` get handled? It seem in the rails-Repo it's getting deleted in the travis run, which is kind of odd...

**Comment by rafaelfranca** [comment]
It should be handled in the same way. The lock file is there to make sure a developer can clone this repository and run the tests and get the same results as everyone. We need to delete on CI to make sure we always test this repository against the latest version of the dependencies.

**Comment by mmoll** [comment]
Added Gemfile.lock removal to travis :green_heart:

**Review by rafaelfranca — APPROVED** [review]

**Comment by mmoll** [comment]
@rafaelfranca Thanks for the patience and merging. A new gem release with this would be nice. :rocket:

**Comment by serkisco** [comment]
Is this going to updated to work with Rails 6.0.0.rc1? I had to go back to 5.2.3 unfortunately because this gem, that I need, didn’t work. Any update would be great please.

**Comment by mmoll** [comment]
@limitlessmind Does it work when using the current master branch of this gem?

**Comment by tomhughes** [comment]
Now that rails 6 is out it would be good to have a new release with this included...

**Comment by mmoll** [comment]
Anybody?

**Pull request merged**